### PR TITLE
openstack: add support to test DPDK on Mellanox hardware

### PR DESCRIPTION
**Pull request by EmilienM**
This will separate the tests between Intel & Mellanox, so both can test
DPDK.

This adds a new workflow for testing DPDK only. We haven't made
SR-IOC working yet with our Mellanox device and we're working on it. In
the meantime, we only need to test DPDK.

The workflow will do the following:
* Create a performanceprofile
* Prepare the workers for vhostuer and noiommu
* Scale the worker to 1
* Run DPDK tests

### CODE REVIEW

**Comment by EmilienM** [comment]
/hold
if it works, we'll add support for 4.10 and 4.9 as well.

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.10.yaml [hunk 1]
```diff
@@ -51,11 +51,19 @@ tests:
   steps:
     cluster_profile: openstack-vh-mecha-central
     workflow: openshift-e2e-openstack-proxy
-- as: e2e-openstack-nfv
+- as: e2e-openstack-nfv-intel
   interval: 72h
   steps:
     cluster_profile: openstack-nfv
     workflow: openshift-e2e-openstack-nfv
```
Comment (MaysaMacedo) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.10.yaml:58]: Shouldn't the workflow here be` openshift-e2e-openstack-dpdk` to allow testing dpdk with intel? and including the appropriate Networks.

**Pull request merged**